lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -65,6 +67,14 @@ def setup_jdbc_config
     # The amount of seconds to wait to acquire a connection before raising a PoolTimeoutError (default 5)
     config :jdbc_pool_timeout, :validate => :number, :default => 5
 
+    # Timezone conversion.
```

Comment:
these docs do a great job describing the behaviors around timezones, but it would be great to be clearer around what setting this parameter will achieve.